x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -468,8 +468,11 @@ const getDataStreams = async (
   esClient: ElasticsearchClient,
   template: IndexTemplateEntry
 ): Promise<CurrentDataStream[] | undefined> => {
-  const { templateName, indexTemplate } = template;
-  const { body } = await esClient.indices.getDataStream({ name: `${templateName}-*` });
+  const { indexTemplate } = template;
+
+  const { body } = await esClient.indices.getDataStream({
+    name: indexTemplate.index_patterns.join(','),
```

Comment:
Ah I see, we generate the `generateTemplateIndexPattern` once and can re-use here. 👍